package-lock.json

```diff
@@ -78819,8 +78892,7 @@
           "dev": true
         },
         "@mongosh/node-runtime-worker-thread": {
-          "version": "3.3.10",
-          "resolved": "https://registry.npmjs.org/@mongosh/node-runtime-worker-thread/-/node-runtime-worker-thread-3.3.10.tgz",
+          "version": "https://registry.npmjs.org/@mongosh/node-runtime-worker-thread/-/node-runtime-worker-thread-3.3.10.tgz",
           "integrity": "sha512-kwFsBy7VQw3LVWZDd8WTp3/29lielff9W6evZqgWCYLzuiaLIO168A7KIRGuvbjYssULCb5+SrUVVYclITEXzg==",
```

Comment:
Don't know why npm install did not update the version here, but instead choose to remove it for this package

Reply:
fixed it, by removing and reinstalling these packages